utils/release_helper.py

```diff
@@ -0,0 +1,790 @@
+        except Exception as e:
+            print(f'Error: {e}')
+            exit(1)
+    else:
+        print(f'Aborted! {milestone} milestone not closed.')
+
+
+def create_repo_milestone(repo, name) -> None:
+    if get_version_milestone(name):
+        print(f'Great! The "{name}" milestone is already created.')
+        return
+
+    latest_release = get_latest_release(repo)
+    estimated_release_date = get_next_release_date(latest_release.published_at)
+    if get_confirmation(f'Are you sure about creating the "{name}" milestone?'):
+        try:
+            repo.create_milestone(
+                title=name, description=f'Milestone for the release {name}',
+                due_on=estimated_release_date)
+        except Exception as e:
+            print(f'Error: {e}')
+            exit(1)
+    else:
+        print(f'Aborted! {name} milestone not created.')
+
+
+# Repository Releases
+def get_repo_releases(repo) -> list:
+    return repo.get_releases()
+
+
+def get_latest_release(repo) -> object:
+    releases = get_repo_releases(repo)
+    # The API returns the list already ordered by published date.
+    return releases[0]
+
+
+# Repository Versions
+def extract_version_from_tag_name(version) -> str:
+    return version.split("v")[1]
+
+
+def get_latest_version(repo) -> str:
+    release = get_latest_release(repo)
+    return extract_version_from_tag_name(release.tag_name)
+
+
+# Repository Next Release
+def bump_version(current_version, bump_position) -> str:
+    # version format example: 0.1.64
+    if bump_position == 'minor':
+        position = 2
+    elif bump_position == 'major':
+        position = 1
+    else:
+        position = 0
+
+    split_version = current_version.split('.')
+    bumped = int(split_version[position]) + 1
+    split_version[position] = str(bumped)
+    return ".".join(split_version)
+
+
+def bump_master_version(repo):
+    content_changed = False
+    old_version = get_next_release_version(repo)
+    new_version = bump_version(old_version, 'minor')
+    old_minor_version = old_version.split('.')[2]
+    new_minor_version = new_version.split('.')[2]
+    repo_root = get_repo_root_path()
+    with open(os.path.join(repo_root, "CMakeLists.txt"), mode='r', encoding='utf-8') as file:
+        content = file.readlines()
+
+    for ln, line in enumerate(content):
+        if f'set(SSG_PATCH_VERSION {old_minor_version}' in line:
+            content[ln] = content[ln].replace(old_minor_version, new_minor_version)
+            content_changed = True
+
+    if content_changed:
+        with open(os.path.join(repo_root, "CMakeLists.txt"), mode='w', encoding='utf-8') as file:
+            file.writelines(content)
+        show_git_diff()
+    else:
+        print('Great! The version in CMakeLists.txt file is already updated.')
+
+
+def get_friday(date) -> datetime:
+    return date + timedelta((4-date.weekday()) % 7)
+
+
+def get_monday(date) -> datetime:
+    return date + timedelta((0-date.weekday()) % 7)
```

Comment:
I wonder why nothing complains about no spaces around minus. Any ideas? I think that they should be there.